client/state/reader/conversations/follow-status.js

```diff
@@ -0,0 +1,9 @@
+/** @format */
+/**
+ * Follow status constants
+ */
+export default {
```

Comment:
should use named exports instead of exporting a default object. that'll allow webpack to catch cases where we try to import the wrong name

Reply:
Good call. I've changed it to follow the same style as `action-types.js`.